app/kafka/manager/actor/cluster/KafkaStateActor.scala

```diff
@@ -397,8 +413,11 @@ trait OffsetCache extends Logging {
       }.groupBy(_._1)
     }
 
-    def getSimpleConsumer(bi: BrokerIdentity) =
-      new SimpleConsumer(bi.host, bi.port, getSimpleConsumerSocketTimeoutMillis, 256 * 1024, clientId)
+    def getSimpleConsumer(bi: BrokerIdentity) = {
+      require(bi.nonSecure, "Cannot fetch log size without PLAINTEXT endpoint!")
```

Comment:
if PLAINTEXT is not provided, then what is the alternative to the the lagSize?

Reply:
I have same question.